Makefile

```diff
@@ -14,39 +15,30 @@ update_config=_config/update.yml
 scrape_config=_config/scrape.yml
 search_config=_config/search_index.yml
 
-# This target is invoked by a doc_independent job on the ROS buildfarm.
-html: build deploy
-
-build: download-previous discover update scrape search-index
+build: prepare-sources
+	bundle exec jekyll build --verbose --trace -d $(site_path) --config=$(config_file),$(index_file)
 
-download-previous:
+prepare-sources:
 	mkdir -p $(remotes_dir)
 	vcs import --input $(remotes_file) --force $(remotes_dir)
 
-discover:
-	bundle exec jekyll build --verbose --trace -d $(deploy_dir) --config=$(config_file),$(index_file),$(discover_config)
-
-update:
-	bundle exec jekyll build --verbose --trace -d $(deploy_dir) --config=$(config_file),$(index_file),$(update_config)
+discover: prepare-sources
+	bundle exec jekyll build --verbose --trace -d $(site_path) --config=$(config_file),$(index_file),$(discover_config)
 
-scrape:
-	bundle exec jekyll build --verbose --trace -d $(deploy_dir) --config=$(config_file),$(index_file),$(scrape_config)
+update: prepare-sources
+	bundle exec jekyll build --verbose --trace -d $(site_path) --config=$(config_file),$(index_file),$(update_config)
 
-search-index:
-	bundle exec jekyll build --verbose --trace -d $(deploy_dir) --config=$(config_file),$(index_file),$(search_config)
+scrape: prepare-sources
+	bundle exec jekyll build --verbose --trace -d $(site_path) --config=$(config_file),$(index_file),$(scrape_config)
 
-# deploy assumes build has run already
-deploy:
-	cd $(deploy_dir) && git add --all
-	cd $(deploy_dir) && git status
-	cd $(deploy_dir) && git commit -m "make deploy by `whoami` on `date`"
-	cd $(deploy_dir) && git push --verbose
```

Comment:
Remove / update deploy instructions on the README

Reply:
Done.